README.md

```diff
@@ -332,7 +332,14 @@ The IP configuration supports the following options:
     Manual DNS configuration can be specified via a list of addresses
     given in the `dns` option, a list of domains to search given in the
     `dns_search` option and a list of dns options to set given in the
-    `dns_options`.
+    `dns_options`. The list of available `dns options` is the same as
+    described in man 5 resolv.conf. The currently supported dns options
```

Comment:
Please use a bullet list for the options to make this easier to read.

Reply:
updated, thanks